pkg/portal/ssh/proxy.go

```diff
@@ -0,0 +1,334 @@
+package ssh
```

Comment:
All this file is :exploding_head: . Would appreciate some comments and a walkthrough again :/